submissions/denismaysov/multiPopup/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MultiPopup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+<div class="page__body">
+    <header class="header">
+        <div class="logo">
+            <a href="https://github.com/denismaysov" target="_blank">
+            <img src="img/user.svg" alt="#"></a>
+        </div>
+        <div class="search">
+            <input placeholder="Search.." class="search__input" id="search" type="search">
+            <label class="search__label" for="search">
+                <button class="search__button" type="submit">
+                    Поиск
+                </button>
+            </label>
```

Comment:
What error here, i checked but i dont understand

Reply:
![image](https://user-images.githubusercontent.com/16196199/187059066-63a6848c-984a-4532-a00f-c5d642352aeb.png)

Reply:
I checked error, but dont understand what i must do. Give ID to button?

Reply:
According to the error, you should add the `id` attribute to the label's children with the same value as the label's attribute `for`.
Or in your case label is redundant. Wrap input and button in the `form` tag. And it will be enough)

Reply:
Still not fixed :cry: